app/config.py

```diff
@@ -241,6 +241,11 @@ class Config(object):
             'task': 'check-job-status',
             'schedule': crontab(),
             'options': {'queue': QueueNames.PERIODIC}
+        },
+        'daily-stats-template_usage_by_month': {
+            'task': 'daily-stats-template_usage_by_month',
+            'schedule': crontab(hour=00, minute=50),
```

Comment:
minor thing - i'd prefer this to be `0` rather than `00` - due to poor decisions, in python 2 a `0` prefix referred to an octal number (it's now `0o123`), but in python3 it's now a syntax error - try typing `01` into a python shell for example. `00` works for some reason /shrug. So I'd prefer this to be just `0`, to dissuade people from trying to prefix everything with 0